static/bidder-params/unicorn.json

```diff
@@ -9,7 +9,7 @@
       "description": "In Application, if placementId is empty, prebid server configuration id will be used as placementId."
     },
     "publisherId": {
-      "type": "integer",
+      "type": "string",
```

Comment:
> it will invalidate currently valid requests and likely cause pain for your publishers
Yes this breaks current specs on our adapter. But we are trying the first integration this time, so this change does not affect to any publisher.